source/AI.cpp

```diff
@@ -3415,11 +3420,16 @@ void AI::AutoFire(const Ship &ship, FireCommand &command, bool secondary, bool i
 	if(ship.IsYours())
 	{
 		auto it = orders.find(&ship);
-		if(it != orders.end() && it->second.target.lock() == currentTarget)
+		if(it != orders.end())
```

Comment:
What's the purpose of breaking up this if statement? It doesn't look to have any effect to me.

Reply:
must have had an else initially